### PR TITLE
setup browserify

### PR DESCRIPTION
**Pull request by chekuda**
- [x] Set up browserify
- [x] Create a bundle file
- [x] Read the bundle file from index.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "body-parser": "^1.17.1",
     "bootstrap": "^3.3.7",
     "express": "^4.15.2"
+  },
+  "devDependencies": {
+    "babel-preset-es2015": "^6.24.0",
```
Comment (Esperteyu) [inline package.json:19]: If we are still bot using react I will wait to use the preset for it. Would do that as a rule of thumb (same with the urlencoded thing in the body-parser)

Comment (Esperteyu) [reply package.json:19]: Yet not*

File: public/scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    request.onreadystatechange = function () {
+      if (request.readyState == 4 && request.status == 200) {
+        checkSuccess(JSON.parse(request.responseText)); // Another callback here
+      }
+    };
+    request.send(JSON.stringify(data));
+  })
+}
+
+runLogin();
```
Comment (Esperteyu) [inline public/scripts/main.js:29]: Can we use an iffe?

**Review by Esperteyu — APPROVED** [review]

File: public/scripts/main.js (hunk 2 above)
Comment (chekuda) [reply public/scripts/main.js:29]: Yes I think thats a good idea. Just choose this option

**Pull request merged**